modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/java/com/liferay/portal/workflow/kaleo/designer/web/internal/portlet/display/context/KaleoDesignerDisplayContext.java

```diff
@@ -395,7 +394,7 @@ public String getMessageKey(
 			return StringPool.BLANK;
 		}
 		else if (workflowDefinitionLinks.size() == 1) {
-			return "workflow-in-use-remove-assignement-to-x-x";
+			return "workflow-in-use-remove-assignement-to-x";
```

Comment:
Hey @MariMalta , Did you check if the `workflow-in-use-remove-assignement-to-x-x` is used in others places? If no, please remove it since it won't be used anymore. 

Thx

Reply:
It's not used in other places, i'm going to remove it. thx =)